backend/webapi/appsettings.json

```diff
@@ -35,7 +35,7 @@
     "Url": "https://localhost:7215/api"
   },
   "KafkaCluster": {
-    "BoostrapServers": "dems-kafka-ccobnjnq-o-c---gs--g.bf2.kafka.rhcloud.com:443",
+    "BoostrapServers": "dems-kafka-cd--b-mj--cba-h----a.bf2.kafka.rhcloud.com:443",
```

Comment:
I've got the local kafka server setup working - so I'll add an appsettings.Development in a PR probably early next week